File: ojuz/ioi/2015/Day1/scales/scales.java
public class scales {
	public void init(int T) {
	}

	public void orderCoins() {
		int[] W = new int[6];
		for (int i = 0; i < 6; i++) {
			W[i] = i + 1;
		}
		grader.lib.answer(W);
	}
}

File: ojuz/ioi/2015/Day1/scales/scales.pas
unit scales;

interface

procedure init(T : longint);
procedure orderCoins();

implementation

uses graderlib;

procedure init(T : longint);
begin
end;

procedure orderCoins();
var i : longint;
    W : array[0..5] of longint;
begin
	for i := 0 to 5 do W[i] := i + 1;
	answer(W);
end;

end.